modules/core/src/main/scala/org/scalasteward/core/sbt/package.scala

```diff
@@ -45,13 +44,8 @@ package object sbt {
       """addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.11")"""
     )
 
-  val stewardPlugin: FileData = {
+  def stewardPlugin[F[_]](implicit fileAlg: FileAlg[F], F: Functor[F]): F[FileData] = {
     val name = "StewardPlugin.scala"
-    // I don't consider reading a resource as side-effect,
```

Comment:
Changed your mind? :D

Reply:
It seems I did. :-) I guess it was convenient to call `unsafeRunSync` here when this code was added. Now that `FileAlg` has `readResource` it seemed more convenient to properly use it.